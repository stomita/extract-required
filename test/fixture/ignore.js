var aaa = require('aaa');
var bbb = require('bbb');
var ddd = require('ddd');
var foo = require('./dir1/foo');
var bar = require('./dir1/dir2/bar');
var baz = require('./dir3/baz');

function example() {
  return baz(aaa() + foo(), bbb() + bar()) * ddd();
}

module.exports = example;
