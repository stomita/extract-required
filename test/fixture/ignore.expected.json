["aaa","bbb","./dir3/baz"]